packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -415,6 +415,20 @@ export default class Meetings extends WebexPlugin {
     }
   }
 
+  /**
+     * API to enable or disable TURN discovery
+     * @param {Boolean} enable
+     * @private
+     * @memberof Meetings
+     * @returns {undefined}
+    */
+  _toggleTurnDiscovery(enable) {
```

Comment:
Because we don't want other clients of SDK to use it. It's a feature that's not rolled out to production yet and only Cantina users with specific feature toggles will be able to use it (and for now also the backend toggles for this will be enabled only on web-int.webex.com)

Reply:
BTW, this is not a new approach that I invented, I'm just following the same pattern from `_toggleUnifiedMeetings()` and `_toggleAdhocMeetings()`

Reply:
I see, thanks for the explanation